website/docs/release_notes.md

```diff
@@ -5,6 +5,178 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.13](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.13)
+
+## Change default command to `repl` when no args are passed
+
+The `run` command is now no longer the default when no arguments are passed, starting from `0.1.13` Scala CLI run scala `repl` under the hood.
+
+```
+$ scala-cli -S 3
+Welcome to Scala 3.1.3 (17.0.3, Java OpenJDK 64-Bit Server VM).
+Type in expressions for evaluation. Or try :help.
+
+scala> 
+```
+
+When inputs are provided, scala-cli uses the `run` command as default.
+
+```
+$ cat hello.sc
+println("Hello World")
+$ scala-cli hello.sc
+Hello World
+```
+
+This changed was added by [@Gedochao](https://github.com/Gedochao) in [#1268]( https://github.com/VirtusLab/scala-cli/pull/1268)
+
+## Optional project settings file
+
+Scala CLI now adds support for the optional configuration file `project.settings.scala` which determines the root workspace directory. The settings file is recommended place to put most using directives (dependencies, scala version, options).
+
+```
+$ cat project.settings.scala
+//> using scala "2.13.4"
+$ cat hello.sc
+println(util.Properties.versionString)
+$ scala-cli hello.sc .
+version 2.13.4
+```
+
+To see how root directory is resolved see [this document](https://github.com/VirtusLab/scala-cli/blob/932c942b78bc35fc0906f2f9e2f6a0c56bef712b/website/docs/reference/root-dir.md)
+
+Added in [#1260]( https://github.com/VirtusLab/scala-cli/pull/1260) by [@wleczny](https://github.com/wleczny)
+
+## Scala CLI is built using Scala 3.2.0
+
+We now rely on Scala `3.2.0` as the default internal Scala version used to build the project.
+
+This changed was added by [@lwronski](https://github.com/lwronski) in [#1314](https://github.com/VirtusLab/scala-cli/pull/1314)
+
+## Add resources support to Scala Native
+
+Scala CLI now allows embedding resources (by default) in a Scala Native binary for `--native` flag.
+
+```
+$ cat resources/scala-native/foo.c 
+int foo(int i) {
+  return i + 42;
+}
+$ cat hello.scala 
+//> using platform "native"
+//> using resourceDir "resources"
+
+import scalanative.unsafe.*
+
+@extern
+def foo(int: CInt): CInt = extern
+
+@main def main =
+  println(foo(3))
+$ scala-cli hello.scala --native 
+45
+```
+
+Added in [#812]( https://github.com/VirtusLab/scala-cli/pull/812) by [@jchyb](https://github.com/jchyb)
+
+##  Compact for `-e` and `--scalac-help`
+
+As we now default to the `repl` sub-command when no args are passed, the `-e` option requires special handling to backward compatible with current scala commands.
+
+```
+$ scala-cli -e 'println("Hello")'
+Hello
+```
+
```

Comment:
````suggestion

If you still want to pass a snippet to the `repl`, you can either pass the `repl` sub-command explicitly or use one of the following options, as before: `--script-snippet`, `--scala-snippet` or `--java-snippet`.

```
$ scala-cli --script-snippet 'println("Hello")'
Welcome to Scala 3.1.3 (17.0.2, Java OpenJDK 64-Bit Server VM).
Type in expressions for evaluation. Or try :help.
                                                                                                                                                             
scala> snippet_sc.main(Array.empty)
Hello
```

This change was introduced to make the `-e` option backwards compatible with the `scala` command.
````